posthog/api/test/test_insight.py

```diff
@@ -455,10 +516,39 @@ def test_insight_items_on_a_dashboard_ignore_deleted_dashboards(self) -> None:
         new_dashboard_id, _ = self.dashboard_api.create_dashboard({})
         # accidentally include a deleted dashboard
         _, update_response = self.dashboard_api.update_insight(
-            insight_id, {"dashboards": [dashboard_id, deleted_dashboard_id, new_dashboard_id]}
+            insight_id,
+            {"dashboards": [dashboard_id, deleted_dashboard_id, new_dashboard_id]},
         )
         assert sorted(update_response["dashboards"]) == [dashboard_id, new_dashboard_id]
 
+    def test_insight_items_on_a_dashboard_ignore_deleted_dashboard_tiles(self) -> None:
```

Comment:
the change in this file only adding this test

Reply:
what black version are you on?